assets/js/graphql/get-organisms.js

```diff
@@ -90,3 +90,81 @@ export function getOrganismsFormDataFunction(queryData={}, pageData={}, options=
   return getOrganisms(queryData, pageData, options)
     .then(({data}) => organismsDataToFormData(data));
 }
+
+
+/** The GraphQL query used to get organisms for trait search. */
+export const getTraitOrganismsQuery = `
```

Comment:
The trait-specific functions in `get-organisms.js` are unnecessary. Just use the existing generic functions and the trait association search component will ignore the `strains` attributes.